apps/web/playwright/organization/booking.e2e.ts

```diff
@@ -324,6 +324,8 @@ test.describe("Bookings", () => {
       const username = "john";
       const userInsideOrganization = await users.create({
         username,
+        useExactUsername: true,
+        email: "john-inside@example.com",
```

Comment:
Due to not using `useExactUsername` it was generating different username when the test was expecting to have same usernames 🤦‍♂️